src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+
+### Data availability vs. data retrievability {#data-availability-vs-data-retrievability}
+
+Data availability is a different idea from data retrievability. Data availability is the ability of peer-to-peer nodes to download transaction data for a block while it is being proposed for addition to the chain. In other words, data availability is relevant when a block is yet to pass consensus. 
+
+Data retrievability is the ability of nodes to download transactions from a block *after* it has passed consensus. Essentially, data retrievability is concerned with retrieving data from historical blocks. A block becomes “historical” when other blocks have been built on it. 
+
+The core Ethereum protocol is primarily concerned with data availability, not data retrievability. It isn’t the blockckhain’s duty to store data for every transaction it reaches consensus on. Besides, storing state data in perpetuity increases hardware requirements for full nodes and may harm decentralization. 
+
+Nevertheless, data retrievability is a much easier problem to solve than data availability. The ability to retrieve historical blockchain data only needs a weak `1-of-N` assumption, meaning only one honest node has to store the data to make it retrievable. It is also assumed that many entities, such as blockchain explorers, have incentives to store archival data and can make it available to others on request. 
+
+[More on solutions to the data retrievability problem](https://notes.ethereum.org/@vbuterin/data_sharding_roadmap#Who-would-store-historical-data-under-sharding).
+
+## Why is data availability important? {#why-is-data-availability-important}
+
+### Blockchain security {#blockchain-security}
+
+Data availability is crucial for blockchain security, or else “data withholding attacks” would become commonplace. A data withholding attack occurs when a miner or validator publishes a block but doesn’t share the transaction data used to build the block. 
+
+If a data withholding attack happens, full nodes cannot verify the correctness of updates to Ethereum’s world state. This gives malicious block producers leeway to subvert protocol rules and advance invalid state transitions on the Ethereum network. 
+
+This is important because other network participants, such as light clients, rely on full nodes to verify the network’s state. Unlike full nodes, light clients only check block headers and don’t download block data. Therefore, the rules around data availability ensure full nodes can validate blocks and prevent the chain from getting corrupted. 
+
+### Decentralized scalability {#decentralized-scalability}
```

Comment:
I'd love to see an explicit section on monolithic vs modular blockchains. I think we imply it a lot but explaining how something like explaining monolithic blockchains do not have a data availability problem because every node replicates all data of every block, but in order to scale without increasing node requirements (decentralisation) we need to go modular. 

Not 100% sure where in this article that would best live, I'd leave that to your interpretation/vision.

Reply:
So, I weaved the distinction between modular and monolithic blockchains into the off-chain DA vs on-chain DA section. That sounded better than creating a new section for the topic.